Select sha1_nacl.cc, md5_nacl.cc implementation.
NACL doesn't have boringssl either, we can just select NACL implementation for
MD5/SHA1 used in upstream Chromium.


diff --git a/base/hash/sha1.h b/base/hash/sha1.h
--- a/base/hash/sha1.h
+++ b/base/hash/sha1.h
@@ -14,11 +14,11 @@
 #include "base/containers/span.h"
 #include "base/strings/string_piece.h"
 #include "build/build_config.h"
-#if BUILDFLAG(IS_NACL)
+#if BUILDFLAG(IS_NACL) || 1
 #include "base/hash/sha1_nacl.h"
 #else
 #include "base/hash/sha1_boringssl.h"
 #endif
 
 namespace base {
 
diff --git a/base/hash/md5.h b/base/hash/md5.h
--- a/base/hash/md5.h
+++ b/base/hash/md5.h
@@ -11,9 +11,9 @@
 #include "base/strings/string_piece.h"
 #include "build/build_config.h"

-#if BUILDFLAG(IS_NACL)
+#if BUILDFLAG(IS_NACL) || 1
 #include "base/hash/md5_nacl.h"
 #else
 #include "base/hash/md5_boringssl.h"
 #endif
 
-- 
2.35.0.rc0.227.g00780c9af4-goog

